elife/config/home-deploy-user-postgres-.env

```diff
@@ -1,2 +1,2 @@
 COMPOSE_PROJECT_NAME=postgres
-IMAGE_TAG=9.4.16
+IMAGE_TAG={{ pillar.elife.docker_postgresql.image_version }}
```

Comment:
it is more correct to say this is an `image_tag` rather than an `image_version`, because you could use e.g `11` or `11-alpine` or `11.5-alpine`, etc

Reply:
done